dev/cognitiveservices/data-plane/Language/readme.md

```diff
@@ -0,0 +1,54 @@
+# Cognitive Services Language SDK
+
+This is the AutoRest configuration file the Cognitive Services Language SDK.
+
+> see https://aka.ms/autorest
+
+## Releases
+
+The current preview release of Language is 2022-07-01-preview.
+
+<!-- The current stable release of Language is 2022-05-01. -->
+
+```yaml
+tag: 
+add-credentials: true
+clear-output-folder: true
+openapi-type: data-plane
+directive:
+  - suppress: LongRunningResponseStatusCode
+    reason: The validation tools do not properly recognize 202 as a supported response code.
+```
+
+### Release
+
+``` yaml $(tag) == ''
```

Comment:
This should still use the same format and include a tag value, something like "package_2022-07-01-preview" (check the existing readmes for examples).